packages/gatsby-theme-carbon/src/components/markdown/Markdown.module.scss

```diff
@@ -93,6 +93,26 @@
   color: $carbon--white-0;
 }
 
+//spacing for content within list item
+.list-item > * + :global(.bx--row),
+.list-item > * + :global(.gatsby-resp-image-wrapper),
+.list-item > * + pre {
+  margin-bottom: 1.5rem;
+}
```

Comment:
For reviewers: I wanted to avoid doing `* + *` here bc this spacing shouldn't also affect nested list items. It seemed better to do this (^above styles) than `* + *:not(:global(.bx--list--nested))`.

Reply:
I think I'd actually prefer the second option. That way we don't have to come in and add exceptions for gifs, svg, video, custom components that site's have written etc.